src/windows/audio.c

```diff
@@ -381,16 +393,28 @@ void MTY_AudioReset(MTY_Audio *ctx)
 	}
 }
 
-static bool audio_handle_device_change(MTY_Audio *ctx)
+static HRESULT audio_reinit_device(MTY_Audio *ctx)
 {
-	if (AUDIO_DEVICE_CHANGED) {
-		audio_device_destroy(ctx);
+	audio_device_destroy(ctx);
+
+	// When the default device is in the process of changing, this may fail so we try multiple times
+	HRESULT e = AUDCLNT_E_NOT_INITIALIZED;
+	for (uint8_t x = 0; x < 5; x++) {
```

Comment:
I would make this 5 into a `#define` constant, and likewise the 100 below. if we do this consistently, then a person can change the # of tries and the # of ms between tries, without first engaging the C language / Parsec idiom parser in their brain